src/editor/console/history/__tests__/reducer.test.js

```diff
@@ -0,0 +1,38 @@
+import reducer from "../reducer";
+
+describe("console/history/CLEAR", () => {
+  let state;
+  beforeEach(() => {
+    state = {
+      history: [
+        {
+          content: "Loading Python language plugin",
+          historyId: "wrmhklxskc",
+          historyType: "CONSOLE_MESSAGE",
+          level: "LOG"
+        },
+        {
+          content: "Python plugin ready",
+          historyId: "xqtol9pck6",
+          historyType: "CONSOLE_MESSAGE",
+          level: "LOG"
+        },
+        {
+          content: "1+2",
+          historyId: "7k77hjno1d",
+          historyType: "CONSOLE_INPUT",
+          language: "py"
+        },
+        {
+          historyId: "6316fcv86c",
+          historyType: "CONSOLE_OUTPUT"
+        }
```

Comment:
Probably not necessary to have quite this many console entries for a valid test -- just 2 should be sufficient.

Reply:
I see, agreed that more than two entries is not necessary. I can remove one entry if that's okay post-approval.

Reply:
2a60407d